scm/src/main/java/com/sandboni/core/scm/cli/ProcessRunner.java

```diff
@@ -0,0 +1,62 @@
+
+package com.sandboni.core.scm.cli;
```

Comment:
I think that move it to .scm.resolvers.cli makes more sense

Reply:
This class is not specific to cli. it can run any process

Reply:
ok